internal/keyrotation/keyrotation.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package keyrotation implements the API handlers for running key rotation jobs.
+package keyrotation
+
+import (
+	"fmt"
+	"net/http"
+
+	"go.opencensus.io/trace"
+
+	"github.com/google/exposure-notifications-server/internal/logging"
+	revisiondb "github.com/google/exposure-notifications-server/internal/revision/database"
+	"github.com/google/exposure-notifications-server/internal/serverenv"
+)
+
+// NewRotationHandler creates a http.Handler that manages deletion of
+// old export files that are no longer needed by clients for download.
+func NewRotationHandler(config *Config, env *serverenv.ServerEnv) (http.Handler, error) {
```

Comment:
We don't have a lot of consistency right now, but I'd like us to start moving to the pattern in `internal/export` where we return a server, the handlers hang off the server, and the server produces its own routes ([example](https://github.com/google/exposure-notifications-server/blob/main/internal/export/server.go#L48)). Then `handler` becomes `server` and `ServeHTTP` becomes `HandleRotate` and returns an `http.Handler`. It makes testing and composing things a bit easier.